### DISCUSSION TITLE
如何在透视表中插入迷你图？

### DISCUSSION THREAD

**Discussion by liran19960215** [antvis/S2#2100] [Q&A] [answered]
你好 目前想要在透视表最后一列插入迷你图显示整体数据趋势 基本诉求和下面的issue一致
https://github.com/antvis/S2/issues/1294
看到评论说2022下半年会支持 我在官网的demo上没有找到相关的图表示例 只有react版本有一个趋势表的类型可以支持
https://s2.antv.antgroup.com/zh/examples/case/kpi-strategy/#covid-trend
所以想请问下目前在vue版本下 透视表PivotSheet支持拆入一列展示迷你图吗

**Comment by lijinke666** [comment, marked as answer]
迷你图确实缺少 demo/相关文档, 近期会补充, 迷你图是 `@antv/s2` 这个包内置的, **只要传入的数据是满足要求的**就会自动渲染迷你图, 和什么框架无关, 你可以尝试下, 不过这一块能力比较弱, 还不太成熟, 2.0 的新版本可以支持渲染 G2 之类的第三方图表库

https://s2.antv.antgroup.com/manual/basic/analysis/strategy#:~:text=%3D%3E%20void%3B-,BaseChartData,-%E6%8A%98%E7%BA%BF%E5%9B%BE%E3%80%81%E6%9F%B1%E7%8A%B6

![image](https://user-images.githubusercontent.com/21015895/221861382-058038b1-55d1-418c-868e-5a09ea38b983.png)


![image](https://user-images.githubusercontent.com/21015895/221860091-f9b10c97-3787-4bf0-a242-62f330ac0314.png)

![image](https://user-images.githubusercontent.com/21015895/221861832-6e27db20-73e8-4c55-9f1f-9aff3d8676da.png)

**Comment by liran19960215** [comment]
你好 我看链接里的迷你图demo表类型都是sheetType="strategy" 现在sheetType="PivotSheet"支持吗 还是说一样的传入数据就会展示 和具体的表类型无关？

**Reply by lijinke666** [reply]
恩，只是demo 是用趋势分析表写的，可能造成歧义了，和什么表格类型，框架没关系

**Reply by liran19960215** [reply]
ok感谢！

**Comment by lijinke666** [comment]
https://s2.antv.antgroup.com/examples/custom/custom-cell/#data-cell
